packages/devtools_app/test/legacy_integration_tests/debugger.dart

```diff
@@ -400,12 +400,6 @@ class DebuggingManager {
     return result.cast<String>();
   }
 
-  Future<bool> supportsScripts() async {
```

Comment:
can you ignore changes in legacy_integration_tests/ for now? These tests need to be re-written as real integration tests, and the existing code will be useful for porting these over, even if it is not used for now (some of these tests are disabled or commented out at the moment)

Reply:
no worries - we already have a TODO at the top of the intetgration_test.dart file to rewrite the integration tests: https://github.com/flutter/devtools/issues/1987